common/src/main/java/com/wynntils/features/inventory/ContainerSearchFeature.java

```diff
@@ -75,7 +76,7 @@ public class ContainerSearchFeature extends Feature {
     @SubscribeEvent
     public void onScreenInit(ScreenInitEvent event) {
         if (!(event.getScreen() instanceof AbstractContainerScreen<?> screen)) return;
-
+        if (!(screen.getMenu() instanceof ChestMenu chestMenu)) return;
```

Comment:
Please keep the empty separator line between the "if this and that then return early" and the real code.

Reply:
can you add that to Spotless?